static/js/linkifiers.js

```diff
@@ -17,7 +17,7 @@ function python_to_js_linkifier(pattern, url) {
         // Replace named group with regular matching group
         pattern = pattern.replace("(?P<" + name + ">", "(");
         // Replace named reference in URL to numbered reference
-        url = url.replace("%(" + name + ")s", "\\" + current_group);
+        url = url.replace("{" + name + "}", "\\" + current_group);
```

Comment:
URI Templates are more than just `{foo}`; an example from the RFC is `http://example.com/search{?q,lang}`. We want a real RFC 6570 implementation here, such as [url-template](https://www.npmjs.com/package/url-template).